src/Payments.Mvc/Controllers/PdfController.cs

```diff
@@ -80,7 +80,6 @@ public async Task<ActionResult> Invoice(string id)
 
             // generate pdf
             var report = await _jsReportMvcService.RenderViewAsync(HttpContext, request, RouteData, "Invoice", invoice);
-            var logs = report.Meta.Logs;
```

Comment:
@sprucely I know Scott hates comments, but I'm guessing this was used for debugging. It might be useful just to comment this out. Unless it is actually doing something.